components/search/SearchSelectedFacets.vue

```diff
@@ -49,6 +52,14 @@
 
         return listOfFacets;
       }
+    },
+
+    methods: {
+      // Takes hex value `#000000` and converts it to it's Standard Color Name `Black`
+      colorHexToStandardColorName(label) {
```

Comment:
This should also be made generic for any facet, as with `MoreFacetsDropdownOptions.vue`.